src/hooks/useDebounce.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from "react"
+
+const useDebounce = (onChange, debounce, initialValue) => {
+  const [event, setEvent] = useState(null)
+  const value = event?.target?.value || initialValue
```

Comment:
this looks dangerous. what happens if the user tries to set an empty value? I believe that would fall back to `initialValue`, no? please write a test for this case